### PR TITLE
WIP: feat(invoice): add basis

### PR DESCRIPTION
**Pull request by pepperfm**
Hi everyone!

I want to introduce a new payment feature that implements this api: https://core.telegram.org/bots/api#payments

Recently, in my project, it became necessary to implement payments through a bot.
I'm using your library with great pleasure, so I decided to create my first PR and start development from this endpoint

This part of the code implements a minimum of functionality of `sendInvoice` endpoint, so i need some feedback from you about code style and classes usability:

- Using of `Arr`facade
- Classes naming
- Dto structure (about missing `__construct`declaration)
- Using Dto as parameter in `TelegramChat` model
etc.

Basically, I'll be glad to receive any comments from you about code-style that you use for

### CODE REVIEW

**Comment by fabio-ivona** [comment]
Hi, thanks for your contribution, will check ASAP

**Comment by fabio-ivona** [comment]
Hi @pepperfm we have a payment feature under testing, so I will take your PR into account in order to improve our code, thanks! we'll be out with it in short time, thanks!

**Comment by ah-rahimi** [comment]
Hi @fabio-ivona,
I wanted to follow up on the status of the payment implementation
Has there been any progress?
When will it be released?
I need this and if you want I can help in implementing it

**Comment by fabio-ivona** [comment]
@ah-rahimi I have a working beta, are you available for testing it?

**Comment by ah-rahimi** [comment]
@fabio-ivona No problem. I can test it in the coming days.

**Comment by fabio-ivona** [comment]
in this case, you can use this to test it: `composer require defstudio/telegraph:dev-payments`

let me know if you feel something to report
